Formula/materialized.rb

```diff
@@ -35,6 +35,41 @@ def install
                                "--path", "src/materialized"
   end
 
+  def caveats; <<~EOS
+    The launchd service will use only one worker thread. For improved
```

Comment:
s/"one worker thread"/"one thread for computation to maintain a low footprint for local development"